packages/pds/src/did-cache/index.ts

```diff
@@ -1,17 +1,19 @@
 import PQueue from 'p-queue'
 import { CacheResult, DidCache, DidDocument } from '@atproto/identity'
-import Database from './db'
-import { excluded } from './db/util'
-import { dbLogger } from './logger'
+import { excluded } from '../db/util'
+import { didCacheLogger } from '../logger'
+import { DidCacheDb, getMigrator, getDb } from './db'
 
-export class DidSqlCache implements DidCache {
+export class DidSqliteCache implements DidCache {
+  db: DidCacheDb
   public pQueue: PQueue | null //null during teardown
 
   constructor(
-    public db: Database,
+    dbLocation: string,
     public staleTTL: number,
     public maxTTL: number,
   ) {
+    this.db = getDb(dbLocation)
```

Comment:
Having the storage encapsulated here makes me happy.

Reply:
yeah it's so much better

perfect fit tbh